sci-libs/Rtree/Rtree-0.9.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..8} )
+
+inherit distutils-r1
+
+MY_PN="Rtree"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="R-Tree spatial index for Python GIS"
+HOMEPAGE="https://rtree.readthedocs.io/en/latest/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+LICENSE="MIT"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+S=${WORKDIR}/${MY_P}
+
+RDEPEND="sci-libs/libspatialindex"
+
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
This is handled by *distutils-r1.eclass* and controlled via `DISTUTILS_USE_SETUPTOOLS` variable, rest LGTM. Didn't build/run test, but you can probably commit this on your own?